drivers/syslog/syslog_channel.c

```diff
@@ -92,7 +92,11 @@ static const struct syslog_channel_s g_default_channel =
 
 /* This is the current syslog channel in use */
 
-FAR const struct syslog_channel_s *g_syslog_channel = &g_default_channel;
+FAR const struct syslog_channel_s
+      *g_syslog_channel[CONFIG_SYSLOG_MAX_CHANNELS] =
+        {
+          &g_default_channel
+        };
```

Comment:
remove the extra space at the begin

Reply:
Can you please explain which space do you refer to?

Reply:
Like this:
```
FAR const struct syslog_channel_s *
g_syslog_channel[CONFIG_SYSLOG_MAX_CHANNELS] =
{
    &g_default_channel
};
```

Reply:
done